cluster/examples/kubernetes/ceph/operator.yaml

```diff
@@ -18,6 +18,18 @@ spec:
     - rcc
   scope: Namespaced
   version: v1beta1
+  additionalPrinterColumns:
+    - name: DataDirHostPath
+      type: string
+      description: Directory used on the K8s nodes
+      JSONPath: .spec.dataDirHostPath
+    - name: MonCount
+      type: string
+      description: Number of MONs
+      JSONPath: .spec.mon.count
+    - name: Age
```

Comment:
Looks like Age is already printed by default. Do you need to add this again when defining additionalPrinterColumns?

Reply:
yeah, they're no longer displayed, when using the additional printer columns